plugins/removeUnknownsAndDefaults.js

```diff
@@ -74,13 +74,23 @@ exports.fn = function(item, params) {
         if (
             params.unknownContent &&
             !item.isEmpty() &&
-            elems[elem].content
+            elems[elem] && //make sure we know of this element before checking its children
+            elem !== 'foreignObject'//Don't check foreignObject
         ) {
             item.content.forEach(function(content, i) {
                 if (
                     content.isElem() &&
-                    !content.prefix &&
-                    elems[elem].content.indexOf(content.elem) === -1
+                    !content.prefix && 
+                    (
+                        (
+                            elems[elem].content && // Do we have a record of its permitted content?
+                            elems[elem].content.indexOf(content.elem) === -1
+                        ) || 
+                        (
+                            !elems[elem].content && // we dont know about its permitted content
+                            !elems[content.elem] // check that we know about the element at all
```

Comment:
This check seems to be redundant. It'll be performed on this element by the plugin itself in a next iteration.